src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:46301/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:46301/project/issues?id=sample&pullRequest=6032&issues=AWeeCDDVoszWdyOs8g7e&open=AWeeCDDVoszWdyOs8g7e)

[ISSUE_KEY]: AWeeCDDVoszWdyOs8g7e